packages/sdk/src/types/runtime/lookAt.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { LookAtMode, Vector3, Vector3Like } from "../..";
+import { ZeroGuid } from "../../constants";
+
+export interface LookAtLike {
+    actorId: string;
+    mode: LookAtMode;
+    backward: boolean;
```

Comment:
>backward: boolean; [](start = 4, length = 18)

Interesting, a backward field? Not using the transform rotation as an offset?

Reply:
Not sure using the transform rotation as an offset is a good idea, since any original rotation would cause LookAt to not look at, and that could feel buggy. After rotating an object, you would have to know to zero out the rotation to make LookAt work again. We could send an offset rotation parameter instead. Backward is really just a 180 rotation offset.

Reply:
It would be reasonable to support 90 offset, etc. I think an offset rotation parameter will be better.

Reply:
Going to stick with `backward`, for simplicity. Other rotations can be achieved using an empty object.

---
In reply to: [261451716](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/225#discussion_r261451716) [](ancestors = 261451716)